docs/get-started/quick-overview.rst

```diff
@@ -16,7 +14,7 @@ Instead of porting LVGL to embedded hardware straight away, it's highly
 recommended to get started in a simulator first.
 
 LVGL is ported to many IDEs to be sure you will find your favorite one.
-Go to the :ref:`simulator` section to get ready-to-use projects that can be run
+Go to the :ref:`integration/ide/pc-simulator/simulator on pc [0-1]` section to get ready-to-use projects that can be run
```

Comment:
What does `[0-1]` mean?

Reply:
Just found the answer [here](https://github.com/lvgl/lvgl/pull/5193#issuecomment-1880025879).

> it's the file path without the file extension followed by an additional "/" and then the section title. At the end there is a marker of [0-0]
what this marker means is [section-level] How the sections are counted up is when a level drops. This is done so the names cannot overlap.

It will cause maintenance issues when a new section/level is inserted and we need to updated all the related links in the docs. I would rather accept to take care of not having not have conflicting headings on a page. Or at worst case if there will be some, a link will point to the wrong section of the correct page. So it's a small issue happening rarely.

Reply:
I can remove those identifiers if you like it's not a big deal to do. but there will be naming collisions. I will make an update to the docs to show you another way of addressing the naming collisions.  I still like the generated names even if the `[*-*]` is not used. I had thought about the same thing with the maintenance aspect of it and I didn't care for it either but thought it still needed to be shown. There are 2 other ways of handling it. One is to rename sections that are duplicates. I don't personally care for this way because is removes uniformity of the documentation. There is another way, I will make those changes so you can see what the other way is.